### PR TITLE
Docs/read plugin markdown

### PR DESCRIPTION
**Pull request by markkelnar**
Source git repo for wp-graphql.com. Parse remark .md files and make available in place of files in /docs/ path.

Solves https://github.com/wp-graphql/wp-graphql/issues/1857

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -112,6 +112,27 @@ const plugins = [
       },
     },
   },
+  {
+    resolve: "gatsby-plugin-git-clone",
+    options: {
+      repository: "https://github.com/markkelnar/wp-graphql.git",
```
Comment (markkelnar) [inline gatsby-config.js:118]: Pull docs from here for this demo until merged in. If /docs/ doesn't exist, the gatsby-source-filesystem below here will fail.

Comment (markkelnar) [reply gatsby-config.js:118]: What if while we are transitioning the documentation to github and merging them, we pull in 'develop' branch from the wp-graphql repo?  And once those are merged to master or a release, we can update to use from there?

File: gatsby-node.js [hunk 2]
```diff
@@ -97,6 +106,19 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
       }
     })
   }
+
+  if (markdownDocs.nodes.length) {
```
Comment (jasonbahl) [inline gatsby-node.js:110]: ok, so if we have the same path for a page created in WordPress and markdown, will this override the page created by WordPress? 

That seems to be what I'm seeing, eh? We do have a `faqs` docs entry in both WP and Markdown, but from what I can tell, there's only one page Gatsby is creating and it's the markdown one.

File: gatsby-config.js (hunk 1 above)
Comment (jasonbahl) [reply gatsby-config.js:118]: @markkelnar I like the idea of pulling from develop, or even a specific `/docs` branch as we work on the migration.

File: gatsby-node.js (hunk 2 above)
Comment (markkelnar) [reply gatsby-node.js:110]: I played around with order, creating a page from MarkDown before the allContent nodes and the last page created with the same uri was the winner.  I see https://www.wpgraphql.com/docs/faqs/ which does exist in the WP content.

File: gatsby-config.js (hunk 1 above)
Comment (markkelnar) [reply gatsby-config.js:118]: We can make this happen.

**Comment by jasonbahl** [comment]
@markkelnar I walked through this locally and things are working as expected. Definitely a good start for where we're heading with this! Good work! Excited to get this moving.

**Pull request merged**